src/store/metadata/stub/content/editions/ons-energy-use-fossil-fuels-by-fuel-type-and-industry.json

```diff
@@ -0,0 +1,101 @@
+{
+  "@context": "https://staging.idpd.uk/#ns",
+  "@id": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions",
+  "@type": "hydra:Collection",
+  "title": "Energy use: fossil fuels by fuel type and industry",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions/2023-06",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry",
+      "identifier": "2023-06",
+      "title": "Energy use: fossil fuels by fuel type and industry",
+      "summary": "The UK's fuel use by industry (SIC 2007 group - around 130 categories) and type (coal, natural gas, petrol, diesel oil for road vehicles (DERV), fuel oil, gas oil, aviation fuel and other), 1990 to 2021. The dataset excludes biofuels and waste.",
+      "description": "The UK's fuel use by industry (SIC 2007 group - around 130 categories) and type (coal, natural gas, petrol, diesel oil for road vehicles (DERV), fuel oil, gas oil, aviation fuel and other), 1990 to 2021. The dataset excludes biofuels and waste.",
+      "publisher": "https://staging.idpd.uk/office-for-national-statistics",
+      "creator": "office-for-national-statistics",
+      "contact_point": {
+        "name": "Environment Accounts",
+        "email": "mailto:environment.accounts@ons.gov.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/climatechange",
+        "https://staging.idpd.uk/topics/carbonemissions"],
+      "frequency": "irregular",
+      "keywords": [
+        "climate",
+        "emissions",
+        "environment",
+        "greenhouse-gas",
+        "pollution"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-06-05T09:30:00Z",
+      "modified": "2023-06-05T12:55:00Z",
+      "spatial_resolution":["K02"],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": ["P1Y"],
+      "temporal_coverage": {
+        "start": "1990-01-01T00:00:00Z",
+        "end": "2021-12-31T00:00:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions/2023-06/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/ons-energy-use-fossil-fuels-by-fuel-type-and-industry/editions/2023-06/versions/1",
+          "issued": "2023-06-05T09:30:00",
+          "modified": "2023-06-05T12:55:00"
+        }
+      ],
+      "next_release": "2024-06-30T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "Year",
```

Comment:
I can't see a CSV file with this PR, but would want to make the CSV headers `snake_case`

Reply:
I understand URIs are not required on the output CSV file; I'm still trying to understand how this api-poc works as I have been on leave.

Not sure how it works with this particular dataset - with Standard Industrial Classification (SIC) and a local codelist, but I have left URIs on the Section column of the CSV file. 

I can remove the URIs from the output CSV, but would obviously require re-transforming the dataset. I am open to doing this.
Thanks.

Reply:
Many thanks.
Great; I will get this done shortly.

Reply:
Changes made as directed - 'Section Notation' and 'Industry Section Name'.

Please let me know if I have missed anything. Thanks.